pkg/collectors/virtualservices.go

```diff
@@ -34,6 +36,23 @@ var (
 		[]string{"virtual_service", "namespace", "gateway"},
 		nil,
 	)
+
+	descVirtualServicesHttpMatchInfo = prometheus.NewDesc(
+		"istio_pilot_virtual_service_http_route_info",
```

Comment:
this is called `descVirtualServicesHttpMatchInfo` but the metric is called `istio_pilot_virtual_service_http_route_info`.

Likewise, below they are reversed.  Can we align the names?

Reply:
Thanks for catching.. Fixed